sdk_extensions/jaeger_remote_sampler/src/test/java/io/opentelemetry/sdk/extensions/trace/jaeger/sampler/JaegerRemoteSamplerTest.java

```diff
@@ -106,20 +112,20 @@ public void connectionWorks() {
 
     // verify
     verify(service).getSamplingStrategy(requestCaptor.capture(), ArgumentMatchers.any());
-    Assert.assertEquals(SERVICE_NAME, requestCaptor.getValue().getServiceName());
-    Assert.assertTrue(sampler.getSampler() instanceof RateLimitingSampler);
-    Assert.assertEquals(
+    Assertions.assertEquals(SERVICE_NAME, requestCaptor.getValue().getServiceName());
+    Assertions.assertTrue(sampler.getSampler() instanceof RateLimitingSampler);
```

Comment:
These should be changed to use assertThat().isInstanceOf, the error message we would get from assertTrue would be very unhelpful.

Reply:
Yeah - there are still quite a few places that use legacy junit assertions. Eventually want to migrate those too - but I don't mind doing partial migration when pointed out :)